pkg/guacanalytics/patchPlanning.go

```diff
@@ -142,8 +142,18 @@ func caseOnPredicates(ctx context.Context, gqlclient graphql.Client, q *queueVal
 				return err
 			}
 		}
-	}
+	// two cases one after the other work like an OR statement
```

Comment:
I don't think this is true. https://go.dev/play/p/Adq3QyWdUAy

Reply:
Got it.  I was confusing with java I believe where this may work (https://stackoverflow.com/questions/18412936/or-operator-in-switch-case).

I'll use a comma instead with is Go's syntax for this I believe.